core/tx_list.go

```diff
@@ -241,8 +250,11 @@ func newTxList(strict bool) *txList {
 func (l *txList) Add(tx *types.Transaction) (bool, *types.Transaction) {
 	// If there's an older better transaction, abort
 	old := l.txs.Get(tx.Nonce())
-	if old != nil && old.GasPrice().Cmp(tx.GasPrice()) >= 0 {
-		return false, nil
+	if old != nil {
+		threshold := new(big.Int).Div(new(big.Int).Mul(old.GasPrice(), big.NewInt(100+minPriceBumpPercent)), big.NewInt(100))
```

Comment:
Also, there are explicit tests testing the lower extremes :) https://github.com/ethereum/go-ethereum/pull/14442/files/d67c6fa44d9001e3f484732e44c4a6ec8a7faa3b#diff-59daeee0a5084a262da2e62280567af0R976